week2/README.md

```diff
@@ -13,5 +13,65 @@ This should mostly make sense to you as well, except the body parsing part. So h
 Although most of this was already covered by the JavaScript class, let's refresh our memories on Callbacks.
 > Watch: https://www.youtube.com/watch?v=pTbSfCT42_M
 
+### Something about ES6 I want you guys to know
+You may hear us talking about this "ES6" all the time. ES6 basically means: the latest version of JavaScript. It has a lot of really nice new features that makes life as developer easier. For you guys, you should remember the following:
+> During the NodeJS course, we will teach you some ES6 features, like Fat Arrow. It's *extremely* important to know whether a function comes from ES6 or from an older version of JavaScript. Why? [Because browsers don't support every new feature just yet](http://kangax.github.io/compat-table/es6/). With Node, on the other hand, you can always control which version of Javascript is running, because it's running on your computer, not in the browser. Node Version 6.x that you are running supports most ES6. 
+So in summary: if you're working on the frontend, you probably don't want to use es6 just yet. In backend, type node --version to see which version you are running, and make sure everyone on the team has the same version by adding "engine" to `package.json` like so: 
+
+```js
+"dependencies": {
+  ...
+},
+"devDependencies": {
+  ...
+},
+"engines": {
+  "node": ">=6.5.0" // this means you need 6.5 or higher
+},
+```
 > Read: [Understanding Asynchronous JavaScript Callbacks Through Household Chores
 ](https://medium.freecodecamp.com/understanding-asynchronous-javascript-callbacks-through-household-chores-e3de9a1dbd04#.8ilr4a7aj) _estimated time: ~1 hour_
+
```

Comment:
@pmcalabrese I see that you moved some reading material from week 3 to week 2, but the students are supposed to do week 3 reading next, so should i just move it there or you it here to archive for the next class?